buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -1043,6 +1206,22 @@
     "CliNuGetApiKey": {
       "value": null,
       "isSecret": true
+    },
+    "DistroRid_Ubuntu1704": {
+      "value": "ubuntu.17.04-$(PB_TargetArchitecture)"
+    },
+    "DockerCommonRunArgs_Ubuntu1704": {
+      "value": "--name $(PB_DockerContainerName)$(DockerTag_Ubuntu1704) -v \\\"$(PB_SourcesDirectory):$(PB_GitDirectory)\\\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\\\"$(PB_GitDirectory)\\\" $(DockerImageName_Ubuntu1704)"
+    },
+    "DockerImageName_Ubuntu1704": {
+      "value": "$(PB_DockerRepository):$(DockerTag_Ubuntu1704)"
+    },
+    "DockerTag_Ubuntu1704": {
+      "value": "ubuntu-17.04-debpkg-7968025-20172512022553"
+    },
+    "PB_DebianId_ubuntu1704-x64": {
```

Comment:
I assume this secret value is going to get added where necessary.  Correct?

Reply:
yes, it's added to Key Vault.